pyvista/plotting/renderer.py

```diff
@@ -2750,6 +2750,26 @@ def set_environment_texture(self, texture, is_srgb=False):
         self.SetEnvironmentTexture(texture, is_srgb)
         self.Modified()
 
+    def remove_environment_texture(self):
+        """Remove the environment texture.
+
+        Examples
+        --------
+        >>> from pyvista import examples
+        >>> import pyvista as pv
+        >>> pl = pv.Plotter(lighting=None)
+        >>> cubemap = examples.download_sky_box_cube_map()
+        >>> _ = pl.add_mesh(pv.Sphere(), pbr=True, metallic=0.9, roughness=0.4)
+        >>> pl.set_environment_texture(cubemap)
```

Comment:
Can we copy and show the plotter here and then do the following? I think the before and after of the method's operation can be easily explained.

Reply:
We don't have the ability to copy a plotter. I'd like to be able to show the plot twice with `auto_close=False`, but our scraper doesn't support that.

I think this is the best we can do at this point.